src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -20,16 +20,13 @@
  */
 final class CacheItem implements CacheItemInterface
 {
-    /**
-     * @internal
-     */
-    const CAST_PREFIX = "\0Symfony\Component\Cache\CacheItem\0";
-
-    private $key;
-    private $value;
-    private $isHit;
-    private $expiry;
-    private $defaultLifetime;
+    protected $key;
```

Comment:
I think @stof meant that you could keep it private, since no other classes could possibly exist to access it.